### DISCUSSION TITLE
prevent the user from changing its password

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#247] [Ideas]
Hi,
can we prevent the user from changing its password (optionally) since there is no password complexity policy applied and user now can change its password to 123 and create a big security vulnerability?
Thanks

**Comment by rejetto** [comment]
you can add this in custom html / top
```html
<style>
#change-password { display:none }
</style>
```

**Reply by GeorgeGeorgeGe** [reply]
OK, Good idea.
Thanks
